RocketRL/python/envs/my_collection/my_awesome_env.py

```diff
@@ -0,0 +1,132 @@
+"""This code is adapted from rllib's example of a custom env"""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import numpy as np
+import gym
+from gym.spaces import Discrete, Box
+from gym.envs.registration import EnvSpec
+
+##TO DO: get cfg in here more elegantly 
+#import your cfg file:
+import yaml
+
+#set home directory
+home_dir = '/Users/ninalopatina/gitrepos/Rocket_RL/'
+config_dir = 'config/'
+
+with open(home_dir + config_dir+"config.yml", 'r') as ymlfile:
+    cfg = yaml.load(ymlfile)
```

Comment:
Why do you need to open your configs file again? This should only be processed once at `main.py` and should persist across different function calls.